views/home/index.ejs

```diff
@@ -86,8 +85,47 @@
               >
                 Generate
               </button>
+              <!-- Button to trigger the help modal -->
+              <button
+                id="modal-btn"
+                class="btn btn-outline-secondary btn-m"
+                type="button"
+                aria-label="information"
+              >
+                <i id="info-button" class="fa-solid fa-circle-question"></i>
+              </button>
+              <!-- Help modal -->
+              <div class="modal" aria-modal="true">
+                <div class="modal-content">
+                  <span class="close-btn">&times;</span>
+                  <h1>Help Menu</h1>
+                  <hr />
+                  <p>
+                    You can specify which social media platforms to share with
+                    others by manually selecting the buttons, toggling the
+                    switches, or using the speech recognition feature on the
+                    page. You can enable and disable speech recognition by
+                    clicking on the microphone button and speaking the following
+                    commands into your device's microphone. You can also
+                    customize the dimensions of your QR code and then generate
+                    it by selecting the "Generate" button.
+                  </p>
+                  <br />

```

Comment:
I don't think the bullet points should be so far away from the text. What you could do is make a table that has 2 columns:
Name of column, column description

Example:
| Casual | Selects all of your casual social media accounts |

Reply:
okay, working on it, thank you!

Reply:
what do you think? @sderacy 
<img width="227" alt="Screen Shot 2022-11-28 at 3 05 37 PM" src="https://user-images.githubusercontent.com/57613959/204370624-3ae59844-b8da-4dfa-99ed-708b57a48a08.png">

Reply:
Looks a lot better already, the last thing I'd recommend is having the scroll bar and making the close btn a little bigger and make sure it's sitting in the top rightmost of the page. Right now it's kind of with the content so it doesn't seem like it sits just perfectly.

After all that, I think it's perfect!

Reply:
also, im thinking of changing the color scheme to any one of these. im just not sure what's better, dark background and light text or light background and dark text. i still want this to be independent of the settings functionality but still highly accessible.

<img width="857" alt="Screen Shot 2022-11-28 at 3 10 48 PM" src="https://user-images.githubusercontent.com/57613959/204371551-982125f2-1218-426f-9ef9-4ed324c72296.png">

Reply:
i did everything else but the scroll bar, i will try to get that done tomorrow. i was working on it today, but it would require me to redo how the entire modal looks now. so im gonna need some time to figure this out. but if i cant get it done by tomorrow night, ill let you know!